### ISSUE TITLE
Alpha blending is incorrect in OffsetImage

### ISSUE THREAD

**Issue by Wendly** [matplotlib/matplotlib#5004]
![456](https://cloud.githubusercontent.com/assets/5761889/9545393/0fd78098-4dbc-11e5-8e4c-ce7d6967be9b.png)

The bottom left icon is drawing by figure.figimage (image.FigureImage). It is correct.
The top right icon is drawing by OffsetImage (image.BboxImage). It is incorrect.

What's wrong? 

The version of matplotlib is 1.4.3. The OS is  windows7.

This is test.png
![test3](https://cloud.githubusercontent.com/assets/5761889/9547343/49225ba6-4dcc-11e5-9b82-744409ddc7fa.png)

This is example code.

``` python
import matplotlib
import matplotlib.pyplot as plt
import matplotlib.image as image
from matplotlib.offsetbox import TextArea, DrawingArea, OffsetImage, AnnotationBbox

if __name__ == '__main__':
    matplotlib.rcParams['figure.facecolor'] = 'ffffff'

    fig, ax = plt.subplots()

    ax.set_title('Draw image')
    ax.set_xlim((-2,2))
    ax.set_ylim((-2,2))

    img = image.imread('test.png')
    fig.figimage(img, 0, 0)

    imagebox = OffsetImage(img, zoom=1)
    annotation = AnnotationBbox(imagebox, (-1.5, -1.5), frameon=False, xycoords='data', boxcoords="offset points", pad=0)
    ax.add_artist(annotation)

    plt.show()
```

**Comment by tacaswell** [comment]
Can you please provide a minimal example code which reproduces the bug, what version of mpl you are using, and which backend?

There is not enough information here yet to start debugging this.

**Comment by Wendly** [comment]
I have already updated some information. Thanks for your friendly reminder.

**Comment by tacaswell** [comment]
Thanks.  Can confirm this on almost-current-master as well.

**Comment by WeatherGod** [comment]
Wouldn't alpha blending be different in the two because one has a white
background and the other has a transparent background?

**Comment by anntzer** [comment]
Fixed as of 2.0.2, unless I am mistaken (in which case feel free to request a reopen).
